README.md

```diff
@@ -1,32 +1,47 @@
 # OpenBook V2
 
+A central-limit order-book program based on [Mango V4](https://github.com/blockworks-foundation/mango-v4) and the [previous OpenBook program](https://github.com/openbook-dex/program) (which was a fork of [Serum](https://github.com/project-serum/serum-dex)).
 
 ## License
 
-See the LICENSE file.
+See the [LICENSE file](LICENSE).
 
-The majority of this repo is MIT licensed, but some parts needed for compiling
-the solana program are under GPL.
+The majority of this repo is MIT-licensed, but some parts needed for compiling
+the Solana program are under GPL.
 
 All GPL code is gated behind the `enable-gpl` feature. If you use the `openbook-v2`
 crate as a dependency with the `client` or `cpi` features, you use only MIT
 parts of it.
 
 The intention is for you to be able to depend on the `openbook-v2` crate for
-building closed-source tools and integrations, including other solana programs
-that call into the openbook program.
+building closed-source tools and integrations, including other Solana programs
+that call into the Openbook program.
 
-But deriving a solana program with similar functionality to the openbook program
+But deriving a Solana program with similar functionality to the Openbook program
 from this codebase would require the changes and improvements to stay publicly
 available under GPL.
 
-## Technical details
-This program is based on [Mango V4](https://github.com/blockworks-foundation/mango-v4) and the [previous OpenBook program](https://github.com/openbook-dex/program) (which was a fork of [Serum])
+## Building & testing
 
-The program is using anchor 0.27.0.
+### Pre-requisites
 
+Before you can build the program, you will first need to install the following:
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Solana](https://docs.solana.com/cli/install-solana-cli-tools)
+- [Anchor](https://www.anchor-lang.com/docs/installation) (v0.27.0)
 
-### Submodules
+### Building
 
-After cloning this repo you'll need to init and update its git submodules.
-Consider setting the git option `submodule.recurse=true`.
```

Comment:
I noticed that there are currently no active submodules (unless I did something wrong) - is this planned? If so, I can add these lines back

Reply:
That's right, was my bad. I've just fixed it here https://github.com/openbook-dex/openbook-v2/pull/7

Reply:
So please add back these lines and I'll merge this PR

Reply:
Sure thing, sir